### DISCUSSION TITLE
change the hotkey

### DISCUSSION THREAD

**Discussion by paxon110** [YimMenu/YimMenu#3596] [General]
hello i need urgant help it says to open the mod i must click VK_LBUTTON however i have no idea where it is and i know it does not work so is there anyway i can change the hotkey without oopening the menu?

**Comment by Yimura** [comment]
Remove you settings file, it’s under `%appdata%/YimMenu/settings.json`.

**Comment by Dydax18** [comment]
YO comment tu a fais pour trouver le fichier svp
